products-prepare/linux/yc-image-cleanup.sh

```diff
@@ -394,6 +412,12 @@ function getNonLockedUsers {
     NOPASSWORD_VALUES=$(printf '!\n!!\n*\n!*\n*!\n')
     for USER in $(cat /etc/passwd | getColumn "$SYSDB_DELIMITER" 1); do
         USERPW=$(cat /etc/shadow | getRowByColumnValue "$SYSDB_DELIMITER" 1 "$USER" | awk -F "$SYSDB_DELIMITER" '{print($2)}')
+        if [ "$USERPW" == "" ]; then
+            OS_TYPE=$(getOS)
+            if [ "$OS_TYPE" == "ALT Server" ]; then
```

Comment:
I believe tcb mechanism is not only present on AltLinux. It should be present on Fedora 36+.

Reply:
i could not find a working tcb in modern fedora distros, but it would be really easy to add it here if needed